tests/test_plugin.py

```diff
@@ -116,3 +116,6 @@ def test_pay_plugin(node_factory):
 
     res = l1.rpc.pay(bolt11=inv['bolt11'])
     assert res['status'] == 'complete'
+
+    with pytest.raises(RpcError, match=r'missing parameter'):
```

Comment:
should this be `match=r'missing required parameter'`?

Reply:
from the test logs, seems that this is the case : `AssertionError: Pattern 'missing parameter' not found in 'RPC call failed: method: pay, payload: {}, error: {'code': -32602, 'message': 'missing required parameter: bolt11'}'`

Reply:
Yep, I ran out of time to fix this, but have pushed now...